src/interfaces/server-status.ts

```diff
@@ -0,0 +1,17 @@
+export interface ServerStatus {
+  id: string
+  name: string
+  requests: number
+
+  statusCodeError: Status
+  responseTimeError: Status
```

Comment:
why do we need to have separate `statusCode` and `responseTime`?
What do you want to achieve?

And this status is not for error only, isn't this?

Reply:
It's for separating status code alert with response time alert. And it's to detect is it currently down/not